purepursuit_node:
  lookahead_distance: 0.1
  odom_topic: "/mavros/vision_pose/pose"
  path_topic: "/astar_path_o"
  cmd_vel_topic: "/ctrl_cmd"
  vehicle_max_speed: 1.0

